arangod/Graph/PathManagement/PathValidatorTracer.h

```diff
@@ -51,10 +51,11 @@ class PathValidatorTracer {
   using PathStore = typename PathValidatorImplementation::PathStoreImpl;
 
   PathValidatorTracer(Provider& provider, PathStore& store,
-                      PathValidatorOptions opts);
+                      PathValidatorOptions opts, bool isSatelliteLeader);
```

Comment:
If we move the parameters into options none of this changes are necessary.

Reply:
Removed.